packages/cozy-konnector-libs/libs/linkBankOperations.spec.js

```diff
@@ -285,4 +285,110 @@ describe('linker', () => {
       })
     })
   })
+
+  describe('linking with combinations', () => {
+    describe('getNotLinkedBills', () => {
+
+      test('it returns the bills that are not linked', () => {
+        const linkingResult = {
+          b1: { bill: { _id: 'b1' }, debitOperation: {} },
+          b2: { bill: { _id: 'b2' } }
+        }
+
+        const expected = expect.arrayContaining([linkingResult.b2.bill])
+
+        expect(linker.getNotLinkedBills(linkingResult)).toEqual(expected)
+      })
+
+      test('it returns an empty array if all bills are linked', () => {
+        const linkingResult = {
+          b1: { bill: { _id: 'b1' }, debitOperation: {} },
+          b2: { bill: { _id: 'b2' }, debitOperation: {} }
+        }
+
+        expect(linker.getNotLinkedBills(linkingResult)).toHaveLength(0)
+      })
+    })
+
+    describe('groupBillsByOriginalDate', () => {
+      const bills = [
+        { _id: 'b1', originalDate: '2018-03-10T00:00:00Z' },
+        { _id: 'b2', originalDate: '2018-03-10T00:00:00Z' },
+        { _id: 'b3', originalDate: '2018-03-10T00:00:00Z' },
+        { _id: 'b4', originalDate: '2018-03-15T00:00:00Z' },
+        { _id: 'b5', originalDate: '2018-03-15T00:00:00Z' },
+        { _id: 'b6', originalDate: '2018-03-20T00:00:00Z' },
+        { _id: 'b7', originalDate: '2018-03-20T00:00:00Z' },
+        { _id: 'b8', originalDate: '2018-03-20T00:00:00Z' },
+        { _id: 'b9', originalDate: '2018-03-20T00:00:00Z' },
+        { _id: 'b10', originalDate: '2018-03-30T00:00:00Z' }
+      ]
+
+      test('it groups bills by their originalDate property', () => {
```

Comment:
I think the time should be removed from the key. It can cause some problems because of timezone etc..